src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java

```diff
@@ -173,6 +173,18 @@ private Imp makeImp(Imp imp) {
         return impBuilder.build();
     }
 
+    private static BigDecimal resolveBidFloor(Imp imp, ExtImpOpenx openxExtImp) {
+        final BigDecimal impBidFloor = imp.getBidfloor();
+        final BigDecimal customFloor = openxExtImp.getCustomFloor();
+        if (impBidFloor == null
+                || impBidFloor.compareTo(BigDecimal.ZERO) == 0
+                || impBidFloor.compareTo(BigDecimal.ZERO) < 0
+                && customFloor.compareTo(BigDecimal.ZERO) > 0) {
```

Comment:
What if customFloor is null?
Should we create methods like isValidFlor and use it for both?

Reply:
Extracted checks to method